web-animations-2/Overview.bs

```diff
@@ -323,6 +323,33 @@ descendant effects and custom effects such that the first condition is:
 >     effect</a> or executing any <a>custom effects</a> associated with an
 >     <a>animation effect</a>.
 
+<h4 id='validating-a-css-numberish-time'>Validating a CSSNumberish time</h4>
+
+The procedure to <dfn>validate a CSSNumberish time</dfn> for an input
+value of <var>time</var> is based on the first condition that matches:
+
+    <div class="switch">
+
+    :    If <em>all</em> of the following conditions are true:
+         * The animation is associated with a [=progress-based timeline=], and
+         * <var>time</var> is not a CSSNumeric value with percent units:
+
+    ::    <a>throw</a> a <span class="exceptionname">TypeError</span>.
+    ::    return false;
+
+    :    If <em>all</em> of the following conditions are true:
+         * The animation is not associated with a [=progress-based timeline=],
+           amd
+         * <var>time</var> is a CSSNumericValue:
+         * The units of time are not <a href="https://www.w3.org/TR/css3-values/#time">duration units</a>.
```

Comment:
Links to other specs should use autolinks https://tabatkins.github.io/bikeshed/#autolink-biblio

E.g. This particular case would be `[[CSS3-VALUES#time|duration units]]`

Reply:
Well, that would produce a section link. You probably actually want `<a type lt="<time>">duration units</a>`, to link to the definition of those units whereever they may be.

(You *should* be able to write `<<time|duration units>>`, but I haven't yet landed the fix harmonizing the parsing of all linking shorthands, ugh.)

Reply:
done